src/hooks/useLocationsOfAllTenantsQuery/useLocationsOfAllTenantsQuery.js

```diff
@@ -0,0 +1,52 @@
+import { useQueries } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+  useStripes,
+} from '@folio/stripes/core';
+
+import useConsortiumTenants from '../useConsortiumTenants';
+import { isUserInConsortiumMode } from '../../utils';
+import {
+  CQL_FIND_ALL,
+  LIMIT_MAX,
+  OKAPI_TENANT_HEADER,
+} from '../../constants';
+
+const useLocationsOfAllTenantsQuery = () => {
+  const [namespace] = useNamespace();
+  const stripes = useStripes();
+  const ky = useOkapiKy();
+
+  const { tenants, isLoading } = useConsortiumTenants();
+
+  const tenantIds = tenants.map(tenant => tenant.id);
+
+  const queries = useQueries(tenantIds.map(tenantId => {
+    return {
+      enabled: Boolean(!isLoading && isUserInConsortiumMode(stripes)),
+      queryKey: [namespace, 'tenant', tenantId],

```

Comment:
it doesn't look like a correct key to me, you are fetching locations here, not a tenant